### PR TITLE
Feat 2 get/create post

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review. It includes

1. Get post
2. Create post
3. Logout 
4. BookMark

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -11,6 +11,8 @@ function App() {
         <Route path="/home" element={<Home />} />
         {/* <Route path="/mock" element={<Mockman />} />  */}
```
Comment (ashishkatiyar2404) [inline src/App.js:12]: you can remove this

File: src/App.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import "./App.css";
```
Comment (ashishkatiyar2404) [inline src/App.js:1]: you can add require Auth routes also

File: src/Components/MainFeed/MainFeed.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+.feed {
+  margin: 0 30px;
+  /* width: 100%;
+  flex: 0.6; */
```
Comment (ashishkatiyar2404) [inline src/Components/MainFeed/MainFeed.css:4]: remove commented code

File: src/Components/MainFeed/MainFeed.jsx [hunk 4]
```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import "./MainFeed.css";
+import { InputOption, PostCard } from "../components";
+import { useState } from "react";
+import { useDispatch } from "react-redux";
+import { createPost } from "../../redux/features/post/postThunk";
```
Comment (ashishkatiyar2404) [inline src/Components/MainFeed/MainFeed.jsx:6]: always give one line space between import files and function

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested some changes pls look into them.
Others look good to merge

**Review by shashikantmaurya0007 — APPROVED** [review]
ashish has suggested few changes.rest looks good to be merged:)

**Pull request merged**